core/block_svg.ts

```diff
@@ -1161,7 +1161,14 @@ export class BlockSvg
       .getRenderer()
       .getConstants()
       .getBlockStyle(blockStyleName);
-    this.styleName_ = blockStyleName;
+
+    if (!this.svgGroup_) {
+      throw new Error('SVG group is not defined');
+    }
```

Comment:
Can you remove this? The `svgGroup_` is non-nullable.

Reply:
Done.